### DISCUSSION TITLE
Is dnscrypt-proxy's DoH functionality enabled solely by setting doh_servers = true, or do I need to do something else/more?

### DISCUSSION THREAD

**Discussion by jdrch** [DNSCrypt/dnscrypt-proxy#1740] [Q&A] [answered]
Apologies for the stupid question. I'm just looking at my config file for the 1st time since probably 2018. It would seem that setting `doh_servers = true` enables DoH functionality just as `dnscrypt_servers = true` enables DNSCrypt functionality, but I'd like to be sure.

Can anyone advise?

**Comment by jumpsmm7** [comment, marked as answer]
> 
> 
> Apologies for the stupid question. I'm just looking at my config file for the 1st time since probably 2018. It would seem that setting `doh_servers = true` enables DoH functionality just as `dnscrypt_servers = true` enables DNSCrypt functionality, but I'd like to be sure.
> 
> Can anyone advise?

from my own personal test, your assumptions are correct.

**Reply by jdrch** [reply]
OK, thanks!
